src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1939,11 +1881,13 @@ let () =
    | [| _mina_exe; version |] when is_version_cmd version ->
        Mina_version.print_version ()
    | _ ->
-       let compile_config = Mina_compile_config.Compiled.t in
+       let itn_features =
+         Sys.getenv "MINA_ITN_FEATURES"
```

Comment:
So this is a change in behavior. Previously this value could only come from compile config (induced by the dune profile).

Now there is a new env variable.

I suggest that we retain `Node_config.itn_features` here, for now. At least until the best strategy to handle itn features is decided.

Reply:
Fair enough. The current design will not work to pass in `itn_features` as a cli argument because it effect things before the cli context is loaded. I will revert to the old way for now, but in the future it will need to be dealt with differently than the others